app/ide-desktop/lib/client/shared.ts

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
I thought I caught all of them (there were several hundred comments changed) but I guess this one slipped by because it comes from an existing (but renamed) file.